apps/frontend/src/lib/storage/types/user/userData/index.ts

```diff
@@ -15,6 +15,7 @@ export const UserDataSchema = z.object({
   telegram: nullToUndefined(TelegramDataSchema),
   lanna: nullToUndefined(LannaDataSchema),
   tensionsRating: nullToUndefined(TensionsRatingSchema),
+  attendance: z.array(z.string()).default([]),
```

Comment:
I'm debating on changing this to nullable, I had to add empty arrays in more places than expected.